Gemfile

```diff
@@ -23,7 +23,7 @@ gem "carrierwave", "~> 2.2" # Upload files in your Ruby applications, map them t
 gem "carrierwave-bombshelter", "~> 0.2" # Protect your carrierwave from image bombs
 gem "cloudinary", "~> 1.23" # Client library for easily using the Cloudinary service
 gem "counter_culture", "~> 3.2" # counter_culture provides turbo-charged counter caches that are kept up-to-date
-gem "ddtrace", "~> 0.54.2" # ddtrace is Datadog’s tracing client for Ruby.
+gem "ddtrace", "~> 0.54.2", require: "ddtrace/auto_instrument" # ddtrace is Datadog’s tracing client for Ruby.
```

Comment:
I see `ddtrace/auto_instrument` being recommended on their docs. It looks like it docs was created Dec last year so this might be a newest feature.